extensions/trello/src/TrelloListItem.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+    List,
+    Icon,
+    ActionPanel,
+    OpenInBrowserAction,
+  } from '@raycast/api'
+  import { TrelloResultModel } from './trelloResponse.model'
+  
+  interface TodoListItemProps {
+    result: TrelloResultModel
+  }
+  
+  export const TodoListItem = ({
+    result,
+  }: TodoListItemProps): JSX.Element => {
+    const todo = result
+  
+    return (
+      <List.Item
+        id={todo.id}
+        title={todo.name}
+        // accessoryTitle={todo.due?.toLocaleDateString()}
+        subtitle={todo.desc}
+        icon={Icon.Checkmark}
+        // TODO: How to do the following?
+        // keywords={todo.labels.name}
```

Comment:
I noticed some extensions that seem to popular this value automatically, but with the Trello API, labels are nested objects inside the main card object, not sure how to return all name values of the label objects…

Reply:
```typescript
todo?.labels?.map(label => label.toUpperCase())
```